jschart/index.js

```diff
@@ -3647,7 +3646,6 @@ function load_datasets(chart) {
 
     if (chart.datasets.all.length > chart.dataset_count) {
       console.log('Resizing SVG for chart "' + chart.chart_title + '".');
-      chart.chart.svg.attr("height", get_svg_height(chart));
```

Comment:
This call cannot be removed.  I think you need to leave this in and add a modification of your viewBox attribute.

Reply:
If the viewBox is not updated you cannot realize the updates of the height attribute.  In the below example there are 8 data sets loaded but only 5 are showing up in the legend:

![image](https://user-images.githubusercontent.com/15911116/61792895-624e7080-ade3-11e9-8341-56f468bb7ea1.png)

Reply:
I think this is fixed too.